source/plugin/basics/data/datamanipulator.rst

```diff
@@ -0,0 +1,132 @@
+================
+Datamanipulators
+================
+
+.. note::
+
+    As of writing, the features described in this document are not fully implemented yet.
+
+    If acquiring of a ``DataManipulator`` or a ``DataManipulatorBuilder`` fails, it is probably due to the implementation missing.
```

Comment:
Not sure this is needed. May confuse people unnecessarily with regard to cloning or offering the data to an entity, or getting back an Optional.absent.

If you want to keep it, maybe mention what error (abstract method exception?) would be expected, and to make an issue on the implementation issue tracker they are using, so the team can prioritize implementation order?